File: CP/code/intro/uva_11799.cs
using System;
using System.Linq;
using System.Collections.Generic;


class UVA11799
{
    public static void Main()
    {
        string temp = Console.ReadLine();
        int T = int.Parse(temp);

        int origT = T;
        while (T-- > 0) 
        {
            temp = Console.ReadLine();
            var ints = temp.Split(new string[] {" "}, 
                                 StringSplitOptions.RemoveEmptyEntries)
                            .Select((si) => int.Parse(si)).ToList();
            ints.RemoveAt(0);
            int speed = ints.Max();
            Console.WriteLine($"Case {origT-T}: {speed}");
        }
    }
}

File: CP/code/intro/uva_11799.cc
#include <cstdio>

int main()
{
    int T;
    scanf("%d", &T);

    const int origT = T;
    while (T--) {
        int N;
        scanf("%d", &N);
        int speed = 0;
        while (N--) {
            int c;
            scanf("%d", &c);
            if (c > speed)
                speed = c;
        }

        printf("Case %d: %d\n", origT - T, speed);
    }
}